db/old_migrations/patch-add-sip-fields.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN TRANSACTION;
```

Comment:
This is the old way of writing migrations. Please change this to the newer js files.